jsonnet/rules.jsonnet

```diff
@@ -186,6 +186,16 @@
               severity: 'warning',
             },
           },
+          {
+            expr: 'cluster:alertmanager_routing_enabled:max > 0',
+            alert: 'AlertmanagerReceiversNotConfigured',
+            annotations: {
+              message: 'Alerts are not configured to be sent to a notification system. Configuring an alert route will allow you to be notified when important failures occur.',
```

Comment:
"Configuring an alert route" -> "Configuring a notification receiver"?

Reply:
I think you need at least one route configured to send alerts and while configuring route you need to configure any receiver either way.

Reply:
The confusion for me is that there's always at least one route in the routing tree (the top-level route) but this isn't really important. As @LiliC commented, it might be good to provide guidance. What about?


>Alerts are not configured to be sent to a notification system, meaning that you may not be notified in a timely fashion when important failures occur. Check the OpenShift documentation to learn how to configure notifications with Alertmanager.

Reply:
Ideally we would point to a playbook, but the above msg sgtm. 👍